packages/application-shell/src/components/application-shell/application-shell.spec.js

```diff
@@ -633,3 +637,554 @@ describe('when dispatching a loading notification', () => {
     });
   });
 });
+describe('when clicking on navbar menu toggle', () => {
+  let operations;
+  beforeEach(() => {
+    operations = mocksForMc.createMockOperations();
+    createGraphqlMockServer(xhrMock, {
+      operationsByTarget: { mc: operations },
+    });
+    selectProjectKeyFromUrl.mockReturnValue(
+      operations.FetchLoggedInUser.me.defaultProjectKey
+    );
+  });
+  it('should expand and collapse menu', async () => {
+    const rendered = renderApp();
+    const button = await rendered.findByTestId('menu-expander');
+    fireEvent.click(button);
+    await wait(() => {
+      expect(window.localStorage.setItem).toHaveBeenCalledWith(
+        STORAGE_KEYS.IS_FORCED_MENU_OPEN,
+        false
+      );
+    });
+    fireEvent.click(button);
+    await wait(() => {
+      expect(window.localStorage.setItem).toHaveBeenCalledWith(
+        STORAGE_KEYS.IS_FORCED_MENU_OPEN,
+        true
+      );
+    });
+    // Check that the support link is rendered
+    // Get the nav container, to narrow down the search area
+    const container = await rendered.findByTestId('left-navigation');
+    const navbarRendered = within(container);
+    expect(navbarRendered.queryByText('Support')).toBeInTheDocument();
+  });
+});
+describe('navbar menu links interactions', () => {
+  async function checkLinksInteractions(
```

Comment:
This function now encapsulates the interaction logic of the 3 tests below, so it's less duplication now.